docs/standard/exceptions/how-to-create-localized-exception-messages.md

```diff
@@ -2,7 +2,6 @@
 title: "How to: create user-defined exceptions with localized exception messages"
 description: "Learn how to create user-defined exceptions with localized exception messages"
 author: Youssef1313
-ms.author: ronpet
```

Comment:
I was searching for this specific article in the diff because I'm aware he is in the ms.author but not author. I don't know how I didn't find it the first time 😄 , that's why I wrote my comment.